tsconfig.json

```diff
@@ -4,6 +4,13 @@
 		"declaration": false,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
+		"lib": [
+			"dom",
+			"es5",
+			"es2015.iterable",
+			"es2015.symbol",
+			"es2015.symbol.wellknown"
+		],
```

Comment:
ok, I was just comparing against the first PR raised and didn't notice the issue... it sounds like adding `"types": [ "intern" ]` could save us another round of meta updates after this lands, but it's your call. :)